src/linear_solver_routines/block_row_projection.jl

```diff
@@ -0,0 +1,45 @@
+# This file is pat of RLinearAlgebra.jl
+"""
+    LinSysBlkRowLQ <: LinSysBlkRowProjection
+
+A mutable structure that represents a standard block row projection method.
+
+# Aliases
+- `BlockKaczmarz`
+
+# Fields
+- `alpha::Float64`, a relaxation parameter that should be set between `0.0` and `2.0`
```

Comment:
Docstring should match the symbol in the `struct`

Reply:
This has been corrected